mteb/tasks/Classification/jpn/WRIMEClassification.py

```diff
@@ -0,0 +1,70 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class WRIMEClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="WRIMEClassification",
+        dataset={
+            "path": "shunk031/wrime",
+            "revision": "3fb7212c389d7818b8e6179e2cdac762f2e081d9",
+            "name": "ver2",
+        },
+        description="A dataset of Japanese social network rated for sentiment",
+        reference="https://aclanthology.org/2021.naacl-main.169/",
+        type="Classification",
+        category="s2s",
+        eval_splits=["test"],
+        eval_langs=["jpn-Jpan"],
+        main_score="accuracy",
+        date=("2011-06-01", "2020-05-31"),
+        form=["written"],
+        domains=["Social"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="The dataset is available for research purposes only. Redistribution of the dataset is prohibited.",
+        socioeconomic_status="high",
+        annotations_creators="human-annotated",
+        dialect=None,
```

Comment:
```suggestion
        dialect=[],
```
None means that it is not annotated, [], means annotated and there is no dialect

Reply:
Fixed in #358 and merged.